bfhcafw

```diff
@@ -54,13 +54,21 @@ mcra    Similar to flint, automatically selects binary and PCI conf path to
         run mcra.
 
 bus     Prints the PCI bus number for the ConnectX interface.
+
+Set the environment variable VERBOSE=1 to see more detailed output.
 EOF
 }
 
 err () {
     echo "bfhcafw: $@" >&2
 }
 
+info () {
+    if [ -n "$VERBOSE" ]; then
+        echo "bfhcafw: info: $@" >&2
```

Comment:
Do you really want to send info messages to stderr?

Reply:
Yes, it's meant more for debugging. A few other scripts like bfvcheck use the output of bfhcafw, and doing something like `VERBOSE=1 bfvcheck` would break if info wasn't sent to stderr.

Reply:
Maybe `debug` would be a better name for the function?